vnext/templates/cpp-app/windows/MyApp/MyApp.cpp

```diff
@@ -20,147 +20,58 @@ struct CompReactPackageProvider
 constexpr PCWSTR windowTitle = L"{{ mainComponentName }}";
 constexpr PCWSTR mainComponentName = L"{{ mainComponentName }}";
 
-float ScaleFactor(HWND hwnd) noexcept {
-  return GetDpiForWindow(hwnd) / static_cast<float>(USER_DEFAULT_SCREEN_DPI);
-}
+_Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR /* commandLine */, int showCmd) {
+  // Initialize WinRT.
+  winrt::init_apartment(winrt::apartment_type::single_threaded);
 
-void UpdateRootViewSizeToAppWindow(
-    winrt::Microsoft::ReactNative::ReactNativeIsland const &rootView,
-    winrt::Microsoft::UI::Windowing::AppWindow const &window) {
-  auto hwnd = winrt::Microsoft::UI::GetWindowFromWindowId(window.Id());
-  auto scaleFactor = ScaleFactor(hwnd);
-  winrt::Windows::Foundation::Size size{
-      window.ClientSize().Width / scaleFactor, window.ClientSize().Height / scaleFactor};
-  // Do not relayout when minimized
-  if (window.Presenter().as<winrt::Microsoft::UI::Windowing::OverlappedPresenter>().State() !=
-      winrt::Microsoft::UI::Windowing::OverlappedPresenterState::Minimized) {
-    winrt::Microsoft::ReactNative::LayoutConstraints constraints;
-    constraints.MaximumSize = constraints.MinimumSize = size;
-    rootView.Arrange(constraints, {0,0});
-  }
-}
+  // Enable per monitor DPI scaling
+  SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
 
-// Create and configure the ReactNativeHost
-winrt::Microsoft::ReactNative::ReactNativeHost CreateReactNativeHost(
-    HWND hwnd,
-    const winrt::Microsoft::UI::Composition::Compositor &compositor) {
   WCHAR appDirectory[MAX_PATH];
   GetModuleFileNameW(NULL, appDirectory, MAX_PATH);
   PathCchRemoveFileSpec(appDirectory, MAX_PATH);
 
-  auto host = winrt::Microsoft::ReactNative::ReactNativeHost();
-
-  // Include any autolinked modules
-  RegisterAutolinkedNativeModulePackages(host.PackageProviders());
-
-  host.PackageProviders().Append(winrt::make<CompReactPackageProvider>());
-
+  // DebugBundlePath is used when loading from a bundle server such as metro. If this parameter
+  // is not provided fallback to a combination of JavaScriptBundleFile and BundleRootPath
+  auto reactInstanceSettingsBuilder {
+    winrt::Microsoft::ReactNative::ReactInstanceSettingsBuilder()
 #if BUNDLE
-  host.InstanceSettings().JavaScriptBundleFile(L"index.windows");
-  host.InstanceSettings().BundleRootPath(std::wstring(L"file://").append(appDirectory).append(L"\\Bundle\\").c_str());
-  host.InstanceSettings().UseFastRefresh(false);
+        .BundleRootPath(appDirectory)
```

Comment:
```suggestion
        .BundleRootPath(std::wstring(L"file://").append(appDirectory).append(L"\\Bundle\\").c_str())
```

Reply:
I now see you put it in the `ReactInstanceSettingsBuilder`. End developers may want to put other urls in there, or customize the path in another way. We shouldn't force/hide adding "file://" and "\\Bundle" in the settings builder.